specific_use_cases/DFU_work_in_progress/station_8760s.ipynb

```diff
@@ -0,0 +1,627 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c34af87a-a2b7-4e93-9db9-d013d9a1fbdf",
+   "metadata": {},
+   "source": [
+    "# Hourly climate projections bias-corrected to the location of a weather station\n",
+    "## Access, Examine, and Export\n",
+    "\n",
+    "- Access and export hourly data at station locations (netCDF or csv)\n",
+    "- Examine the “8760’s” and find the one with the smallest distance from the median.\n",
+    "\n",
+    "Start with the usual, and import our python package:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4536d890-1b3b-4db5-9f45-509858a0e6d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import climakitae as ck"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76512298-df93-4568-9ab2-74c7632b8cd8",
+   "metadata": {},
+   "source": [
+    "Optional: launch a cluster. You may wait several minutes, but some subsequent steps will then go faster.\n",
+    "\n",
+    "This is temporarily unadvised when working with station data, but we're trouble-shooting it."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "8289bb39-b992-4428-bba0-6c0aeb3fb4e0",
+   "metadata": {},
+   "source": [
+    "from climakitae.cluster import Cluster\n",
+    "cluster = Cluster()\n",
+    "cluster.adapt(minimum=0, maximum=16)\n",
+    "client = cluster.get_client()\n",
+    "cluster"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2845243-040c-4c83-ac5d-e8f70c8991aa",
+   "metadata": {},
+   "source": [
+    "### Access\n",
+    "#### Start an application and launch the select panel\n",
+    "For the rest of the examples in this notebook, you will want to select 'station' for 'Data type'. Make any other choices you like."
```

Comment:
I tried this for the greater bay area demand forecast zone, and in that zone Oakland Metro and San Francisco international airport stations seemingly had no data (returned all nan's) - not sure if this is a bug or desired behavior (if desired, we may eventually want to implement a warning message or something explaining what's going on for users)

Reply:
That is not an expected behavior...

Reply:
Appears that this is an underlying bug with the data or climakitae code, which has been noted, and should not stand in the way of merging this notebook to have stakeholders test it.